src/power_grid_model/utils.py

```diff
@@ -14,6 +14,7 @@
 
 import numpy as np
 
+from power_grid_model._utils import get_and_verify_batch_sizes, get_batch_size  # pylint: disable=unused-import
```

Comment:
since they are now used: remove `# pylint: disable=unused-import`

in addition, since we actually don't want to expose these functions (we want users to use the `get_dataset_batch_size` instead of `get_and_verify_batch_sizes`:

```suggestion
from power_grid_model._utils import (
    get_and_verify_batch_sizes as _get_and_verify_batch_size,
    get_batch_size as _get_batch_size
)
```

to make them "private". i don't know whether it will work correctly with pylint and isort. If it doesn't work, that's fine as well

Reply:
Isort again 'fixed' this, but to an acceptable state